app.py

```diff
@@ -1,2 +1,57 @@
 from flask import Flask, request
 import json
+
+
+app = Flask(__name__)
+with open("restaurants.json", "r") as read_file:
+    data = json.load(read_file)
+
+restaurants = data['restaurants']
+
+
+@app.route('/api/restaurants', methods=['GET', 'POST'])
+def all_restaurants():
+    if request.method == 'POST':
+        content = request.get_json(silent=True)
+        print(type(content))
```

Comment:
Avoid any `print` statements unless it is a console application where you do need to print something to console. You can have them for debug purposes, during development, but the code in PR should be clean

Reply:
fixed